Testing/kwsBadCharactersTest.cxx

```diff
@@ -15,55 +15,77 @@
  *  limitations under the License.
  *
  *=========================================================================*/
-#if defined(_MSC_VER)
-#pragma warning ( disable : 4786 )
-#endif
-
-#if defined(__clang__)
-#pragma clang diagnostic ignored "-Winvalid-source-encoding"
-#endif
 
 #include "kwsParser.h"
 
 int kwsBadCharactersTest(int, char* [] )
 {
-  std::string buffer;
-  buffer = "void()\n{\ntest����;\n};";
```

Comment:
Interesting to see that github interprets/displays the <E4><FC><F6><DF> sequence as ISO/IEC 8859-1.